crates/bevy_reflect/src/impls/uuid.rs

```diff
@@ -0,0 +1,8 @@
+use crate as bevy_reflect;
+
+use crate::{ReflectDeserialize, ReflectSerialize};
+use bevy_reflect_derive::{impl_from_reflect_value, impl_reflect_value};
+use bevy_utils::Uuid;
+
+impl_reflect_value!(::bevy_utils::Uuid(Serialize, Deserialize));
```

Comment:
Can we also register some other traits? Namely `Default`, `Debug`, `PartialEq`, and `Hash`?

Reply:
Done